app/components/unregistered-contributor-form.js

```diff
@@ -23,7 +23,7 @@ const Validations = buildValidations({
             validator('presence', true),
             validator('format', {
                 type: 'email',
-                regex: /^[-a-z0-9~!$%^&*_=+}{'?]+(\.[-a-z0-9~!$%^&*_=+}{'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|swiss|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i,
+                regex: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
It would be good to put a comment in the code documenting where you found the regex. Why did you go with this one over the one at https://emailregex.com ?

Reply:
It looks like the JS regex in this site and the regex used in the stackoverflow link/this PR are the same. Although it looks like the link you posted escapes the `[`, so that might be better? Either way, I will leave a comment here, as the diagram in your link is very handy!